server/controllers/login.js

```diff
@@ -0,0 +1,131 @@
+const jwt = require('jsonwebtoken')
+const User = require('./../models/user')
+const axios = require('axios')
+const crypto = require('crypto')
+const constants = require('./../config/constants')
+const jsonfile = require('jsonfile')
+
+const inconsistentUsersPath = __dirname + '/inconsistentUsers.json'
+
+
+module.exports.createToken = async function(req, res){
+    try{
+        //Fetch access_token from github
+        const requestToken = req.query.code
+        const ghTokenResponse = await axios({
+            method: 'post',
+            url: `${constants.githubAuthURL}?client_id=${constants.githubClientID}&client_secret=${constants.githubClientSecret}&code=${requestToken}`,
+            headers: {
+                accept: 'application/json'
+            }  
+        })
+        //Check for permission scopes
```

Comment:
@djcruz93 This is the response I get from the above http request
![image](https://user-images.githubusercontent.com/34130764/105610023-13680980-5dd3-11eb-9282-d757c6696c24.png)
As you can see the scope field is empty, however I did agree to all the required permission i.e the followings
![image](https://user-images.githubusercontent.com/34130764/105610054-46aa9880-5dd3-11eb-8a99-3208cba4b5a0.png)

![image](https://user-images.githubusercontent.com/34130764/105610061-54601e00-5dd3-11eb-9a16-7659f7810988.png)

Due to this blank field, the check we have added here isn't passing. Could you retest again wrt this?